src/components/RangeSlider/components/DualThumb/DualThumb.tsx

```diff
@@ -399,6 +402,28 @@ export default class DualThumb extends React.Component<Props, State> {
     }
   }
 
+  @autobind
+  private handleMouseDownTrack(event: React.MouseEvent) {
+    const clickXPosition = this.actualXPosition(event.clientX);
+    if (event.button !== 0) return;
+
+    const {value} = this.state;
+
+    const distanceFromLowerThumb = Math.abs(value[0] - clickXPosition);
+    const distanceFromUpperThumb = Math.abs(value[1] - clickXPosition);
+
+    if (
+      distanceFromLowerThumb < distanceFromUpperThumb ||
+      distanceFromLowerThumb === distanceFromUpperThumb
```

Comment:
I chose to favour the lower thumb if the click is equidistant (for no particular reason).